src/panels/config/automation/condition/types/ha-automation-condition-time.ts

```diff
@@ -44,176 +32,136 @@ export class HaTimeCondition extends LitElement implements ConditionElement {
     return {};
   }
 
-  protected render() {
-    const { after, before, weekday } = this.condition;
+  private _schema = memoizeOne(
+    (
+      localize: LocalizeFunc,
+      inputModeAfter?: boolean,
+      inputModeBefore?: boolean
+    ): HaFormSchema[] => {
+      const modeAfterSchema = inputModeAfter
+        ? { name: "after", selector: { entity: { domain: "input_datetime" } } }
+        : { name: "after", selector: { time: {} } };
+
+      const modeBeforeSchema = inputModeBefore
+        ? { name: "before", selector: { entity: { domain: "input_datetime" } } }
+        : { name: "before", selector: { time: {} } };
+
+      return [
+        {
+          name: "mode_after",
```

Comment:
This one needs a label, it's not clear that this is after:

![image](https://user-images.githubusercontent.com/1444314/154750136-3faa098c-2603-4ca7-8922-7ad8a4ce44fd.png)